src/hooks/useFocusStyle.ts

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+
+export const focusAttributeName = 'data-focus-visible-added';
+
+/*
+ * Adds given className to focusable elements only when focus is triggreded by tab-button.
+ */
+const useFocusStyle = ({
```

Comment:
I was thinking that should the word "tab" be mentioned on the name, like `useTabFocusStyle` as it style is displayed on when traversing with tab.

Reply:
Yea I thought this could have better naming, this one is basically the first thing that came to my mind

Reply:
Updated naming